UnitedIncome/components
# [APP-5099] Paginated Tile

### Description
> Please provide a brief description of your changes below.

Addressed visual bug of pagination arrows being off center.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Verify that the pagination arrows are properly centered and aligned.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [APP-5099](https://unitedincome.atlassian.net/browse/app-5099)
